app/src/main/java/fr/free/nrw/commons/upload/depicts/DepictsContract.java

```diff
@@ -40,6 +43,18 @@
          * add depictions to list
          */
         void setDepictsList(List<DepictedItem> depictedItemList);
+
+        Context getFragmentContext();
```

Comment:
Could you please add javadoc for each added method?
Do not hesitate to copy/paste javadoc from the implementation of each method.
Thanks!